tool/tctl/common/resource_command.go

```diff
@@ -973,6 +986,26 @@ func (rc *ResourceCommand) getCollection(client auth.ClientI) (ResourceCollectio
 			return nil, trace.Wrap(err)
 		}
 		return &lockCollection{locks: []types.Lock{lock}}, nil
+	case types.KindDatabaseServer:
+		ss, err := client.GetDatabaseServers(ctx, rc.namespace)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+		if rc.ref.Name == "" {
+			return &dbCollection{servers: ss}, nil
+		}
+
+		var out []types.DatabaseServer
+		for _, s := range ss {
+			if s.GetName() == rc.ref.Name {
+				out = append([]types.DatabaseServer{s})
```

Comment:
What's this append for if we're returning right after? Should this be outside the "if"?

Reply:
Not sure. If database servers are not distinguishable by the hostID from user perspective wonder if there is any sens returning all HA agents details ?